locale/en_US/api.po

```diff
@@ -96,3 +96,12 @@ msgstr "The public files directory was not found or files can not be saved to it
 
 msgid "api.submissionFiles.400.assocTypeAndIdRequired"
 msgstr "You can not modify the file association without passing an assocType and assocId."
+
+msgid "api.api_key_secret.should.be.filled"
+msgstr "api_key_secret should be filled"
+
+msgid "api.invalid_token_signature"
+msgstr "Could not verify token authenticity. Try again later."
```

Comment:
Let's include the HTTP status code in the msgids and use camel-case instead of `_` or `.` for all three of these. So instead of `api.api_key_secret.should.be.filled`it could be `api.500.apiSecretKeyMissing`. And instead of `api.invalid_token_signature` it could be `api.400.invalidApiToken`. Then sort these messages alphabetically so they're up near the top with the other error codes.

And here is some text to use for each error message:

> The API token can not be used to access this site because the site administrator has not configured a secret key.

> The API token could not be validated. This may indicate an error in the API token or that the API token is no longer valid.